### PR TITLE
(zh_cn) update chinese simplified security page

### PR DESCRIPTION
**Pull request by cursorzz**
Related #1074
screen shot 2015-05-29 at 10 40 39

didnt know how to translate PaaS platformers, any idea? 

also, for more know issues list, I think I should wait until these news zh_cn version updated, then I can start steal their title name to this page 

2015-6-4 update
screen shot 2015-06-04 at 13 01 12

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia @debbbbie

Please help review and think 'PaaS platformers' translation. Thanks!

File: zh_cn/security/index.md [hunk 1]
```diff
@@ -8,7 +8,10 @@ lang: zh_cn
 
 ## 报告安全漏洞
 
-安全漏洞应该通过电子邮件报告到security@ruby-lang.org。这是一个非公开的邮件列表。 报告的问题将在被修正以后才会公开发布。
+安全漏洞应该通过电子邮件报告到security@ruby-lang.org ([the PGP public key](/security.asc))。这是一个非公开的邮件列表。 报告的问题将在被修正以后才会公开发布。
+
+这个邮件列表的成员为Ruby提供支持(包括Ruby代码提交者, 其他Ruby实现的作者, 发布者, 以及PaaS 平台提供者)。
```
Comment (hlcfan) [inline zh_cn/security/index.md]: + Parenthesis.
+ **以及 PaaS 平台提供者**
+ I prefer **PaaS 平台** for *PaaS platformer*

Comment (cursorzz) [inline zh_cn/security/index.md]: well... the PaaS平台 sound not like a human for me, I think those should be human with different jobs or titles

Comment (hlcfan) [inline zh_cn/security/index.md]: Yea, but **Paas 平台** would be a general name for people work for Paas platform.

File: zh_cn/security/index.md [hunk 2]
```diff
@@ -8,7 +8,10 @@ lang: zh_cn
 
 ## 报告安全漏洞
 
-安全漏洞应该通过电子邮件报告到security@ruby-lang.org。这是一个非公开的邮件列表。 报告的问题将在被修正以后才会公开发布。
+安全漏洞应该通过电子邮件报告到security@ruby-lang.org ([the PGP public key](/security.asc))。这是一个非公开的邮件列表。 报告的问题将在被修正以后才会公开发布。
```
Comment (aidewoode) [inline zh_cn/security/index.md]: Need space between Chinese character and English character.

File: zh_cn/security/index.md (hunk 1 above)
Comment (aidewoode) [inline zh_cn/security/index.md]: I think this line need the space between Chinese character and English character too.

File: zh_cn/security/index.md [hunk 3]
```diff
@@ -16,4 +19,6 @@ lang: zh_cn
 
 {% include security_posts.html %}
 
-See [the English page](/en/security/) for prior security related posts.
```
Comment (marocchino) [inline zh_cn/security/index.md]: Don't remove this link. It is intended one. (see 62953b6, #403)

Comment (JuanitoFatas) [inline zh_cn/security/index.md]: :+1:

Comment (JuanitoFatas) [inline zh_cn/security/index.md]: :+1:

File: zh_cn/security/index.md (hunk 1 above)
Comment (cursorzz) [inline zh_cn/security/index.md]: sorry guys, been out for few days, will update them thx

File: zh_cn/security/index.md (hunk 3 above)
Comment (cursorzz) [inline zh_cn/security/index.md]: thx, will fix that

**Comment by stomar** [comment]
**Never merge master or other branches into your topic branch!**

 Please provide a **new PR** with only your proposed change in a **single commit**, e.g. `Update security page (zh_cn)`.

**Comment by JuanitoFatas** [comment]
@cursorzz Please do a rebase from master. If you have any problems, ask here. Thank you.

**Pull request closed without merging**